plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -79,7 +83,7 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
     // TODO Come up with a better way than this. This might not be an issue after CON-1069 since we won't be building
     //  the conclave-build image anymore.
     fun buildInDocker(buildInDocker: Property<Boolean>): Boolean {
-        return !OperatingSystem.current().isLinux || buildInDocker.get()
+        return !OperatingSystem.current().isLinux || buildInDocker.get() || runtimeType.get() == GradleEnclavePlugin.RuntimeType.GRAMINE
```

Comment:
Just add a comment that Gramine enclaves will always be built in the image, whilst GraalVM enclaves will be by default but can be opted out with the `buildInDocker` config.

Reply:
Added